math-magicians/package.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
- [ ] Please move  your project from the folder `math-magician` to the main directory  as shown in the picture: 

![image](https://user-images.githubusercontent.com/58504655/172287297-34ed512b-ce4b-4d10-b2f9-6823c3013535.png)

Reply:
I don't understand the main directory is MATH-MAGICIANS and it is in, isn't it?
![image](https://user-images.githubusercontent.com/58642949/172288783-9a693114-d1a1-4b53-916f-86494b46a260.png)
![image](https://user-images.githubusercontent.com/58642949/172288829-28098d58-fc6d-447e-a303-bfa85e195145.png)

Reply:
> I don't understand the main directory is MATH-MAGICIANS and it is in, isn't it? ![image](https://user-images.githubusercontent.com/58642949/172288783-9a693114-d1a1-4b53-916f-86494b46a260.png) ![image](https://user-images.githubusercontent.com/58642949/172288829-28098d58-fc6d-447e-a303-bfa85e195145.png)

As you can see from the screenshot above, move the content present in math magicians folder to the main directory of the branch. It would look something like this 
![image](https://user-images.githubusercontent.com/58504655/172289327-c66481ff-0600-4c92-9ce4-8ac94a6ac5cf.png)

The thing you did is `create react app myapp` in an existing folder. You could have done ` create react app . `

I hope this makes sense now. Let me know if you have any other questions,

Reply:
Ok, thank you for your help. I will do it.